test/backup.ts

```diff
@@ -529,4 +565,76 @@ describe('Bigtable/Backup', () => {
       );
     });
   });
+
+  describe('setIamPolicy', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+    const policy = {};
+    it('should correctly call Table#setIamPolicy()', done => {
+      sinon
+        .stub(Table.prototype, 'setIamPolicy')
+        .callsFake((_policy, gaxOpts, callback) => {
+          assert.strictEqual(_policy, policy);
+          assert.deepStrictEqual(gaxOpts, {});
+          callback(); // done()
+        });
+      backup.setIamPolicy(policy, done);
+    });
+
+    it('should accept gaxOptions', done => {
+      const gaxOptions = {};
+
+      sinon
+        .stub(Table.prototype, 'setIamPolicy')
+        .callsFake((_policy, gaxOpts, callback) => {
+          assert.strictEqual(_policy, policy);
+          assert.strictEqual(gaxOpts, gaxOptions);
+          callback(); // done()
+        });
+      backup.setIamPolicy(policy, gaxOptions, done);
+    });
+  });
+
+  describe('testIamPermissions', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    const permissions = 'bigtable.tables.get';
+    it('should properly call Table#testIamPermissions', done => {
+      sinon
+        .stub(Table.prototype, 'testIamPermissions')
+        .callsFake((_permissions, gaxOpts, callback) => {
+          assert.strictEqual(_permissions, permissions);
+          assert.deepStrictEqual(gaxOpts, {});
+          callback(); // done()
+        });
+      backup.testIamPermissions(permissions, done);
+    });
+
+    it('should accept permissions as array', done => {
+      const permissions = ['bigtable.tables.get', 'bigtable.tables.list'];
```

Comment:
I think these should be similar to: https://github.com/googleapis/java-bigtable/pull/450/files#diff-84d1a07ba982b6e9cac6f4e6ae0bb75a4531804decf3de85ece5e475c7720801R356-R359

Reply:
(and elsewhere in these tests)

Reply:
Done
Thanks